### PR TITLE
Fix Velopop (Avignon)

### PR DESCRIPTION
**Pull request by Manu1400**
documentation: https://transport.data.gouv.fr/datasets/velos-libre-service-grand-avignon-velopop-disponibilite-en-temps-reel

### CODE REVIEW

File: pybikes/data/gbfs.json [hunk 1]
```diff
@@ -907,6 +907,20 @@
...
+                "city":"Avignon",
+                "name":"V\u00e9lopop",
+                "country":"FR",
+                "company": [
+                    "Smoove"
+                ],
+                "latitude":43.943689,
+                "longitude":4.805833
+            },
+            "feed_url":"https://avignon-gbfs.klervi.net/gbfs/en/station_information.json"
```
Comment (eUgEntOptIc44) [inline pybikes/data/gbfs.json:922]: the `feed_url` should be `https://avignon-gbfs.klervi.net/gbfs/en/gbfs.json` to match the criteria of the [gbfs.py](https://github.com/eskerda/pybikes/blob/97414d1c9f3ebef6cf96e8637b15a77820d32612/pybikes/gbfs.py) parser.

**Review by eUgEntOptIc44 — CHANGES_REQUESTED** [review]

**Comment by eUgEntOptIc44** [comment]
just tested the gbfs feed at [`https://avignon-gbfs.klervi.net/gbfs/en/station_information.json`](https://avignon-gbfs.klervi.net/gbfs/en/station_information.json) and noticed 2 `stations` featuring **invalid** coordinates:

```json
// snippet from (archived) source: https://web.archive.org/web/20221225185948/https://avignon-gbfs.klervi.net/gbfs/en/station_information.json
{
	"station_id": "999_028",
	"name": "Médiathèque",
	"lat": null,
	"lon": null,
	"capacity": 24
}, {
	"station_id": "999_032",
	"name": "Olivades",
	"lat": null,
	"lon": null,
	"capacity": 16
}
```

This returns a `TypeError` when trying to parse to `float` in [gbfs.py](https://github.com/eskerda/pybikes/blob/97414d1c9f3ebef6cf96e8637b15a77820d32612/pybikes/gbfs.py#L145).

**Comment by eskerda** [comment]
> just tested the gbfs feed at [`https://avignon-gbfs.klervi.net/gbfs/en/station_information.json`](https://avignon-gbfs.klervi.net/gbfs/en/station_information.json) and noticed 2 `stations` featuring **invalid** coordinates:
> 
> ```json
> // snippet from (archived) source: https://web.archive.org/web/20221225185948/https://avignon-gbfs.klervi.net/gbfs/en/station_information.json
> {
> 	"station_id": "999_028",
> 	"name": "Médiathèque",
> 	"lat": null,
> 	"lon": null,
> 	"capacity": 24
> }, {
> 	"station_id": "999_032",
> 	"name": "Olivades",
> 	"lat": null,
> 	"lon": null,
> 	"capacity": 16
> }
> ```
> 
> This returns a `TypeError` when trying to parse to `float` in [gbfs.py](https://github.com/eskerda/pybikes/blob/97414d1c9f3ebef6cf96e8637b15a77820d32612/pybikes/gbfs.py#L145).

We can try to filter these... let me think of something. Thanks for checking!

**Comment by eUgEntOptIc44** [comment]
> We can try to filter these... let me think of something. Thanks for checking!

Hi @eskerda,

As noted above I just published my appoarch to solve this in a new PR. Hope you might have the time to provide me with some feedback on this. 
Regards,
Jean-Luc

**Pull request closed without merging**